sync/src/main/java/org/bf2/sync/ManagedKafkaSync.java

```diff
@@ -333,10 +345,18 @@ void create(ManagedKafka remote) {
            if (e.getStatus().getCode() != HttpURLConnection.HTTP_CONFLICT) {
                throw e;
            }
-           log.infof("ManagedKafka %s already exists", Cache.metaNamespaceKeyFunc(remote));
+           // it's possible, but unlikely, that we'll see two reconcilitions close enough together
+           // such that we'll naturally see a conflict - that is the second completes before the cache
+           // is updated.  It's more likely that the cache state is not up-to-date.
+           exit(remote);
        }
     }
 
+    private void exit(ManagedKafka remote) {
+        log.errorf("ManagedKafka %s exists, but not in the informer cache.  The sync is not in a healthy state", Cache.metaNamespaceKeyFunc(remote));
```

Comment:
It might be useful to generate a stack trace for the `errorf` to see which invocation brings us here.